test/next-wepack-compression.test.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const withCompression = require('../next-webpack-compression');
+const withCompression = require('../index');
```

Comment:
I would keep this import pointing to `../next-webpack-compression` since this is the specific test for that file.

The way this module exports its functions is a separate question.